website/source/docs/connect/intentions.html.md

```diff
@@ -111,9 +116,9 @@ valid non-wildcard intentions to match.
 
 The numbers in the table above are not stable. Their ordering will remain
 fixed but the actual number values may change in the future.
-The numbers are non-contiguous because there are
-some unused values in the middle in preparation for a future version of
-Consul supporting namespaces.
+
+-> **Consul Enterprise** - Namespaces are an Enterprise feature. In Consul OSS any of the rows in
+the table with a `*` for either the source or destination namespace are not possible.
```

Comment:
```suggestion
the table with a `*` for either the source namespace or destination namespace are not applicable.
```
Does changing it to "applicable" change the meaning?

Reply:
I don’t think it does.

Reply:
How would you feel about listing the precedences'? Are they likely to change?